miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditSampleController.java

```diff
@@ -981,6 +983,7 @@ protected void writeConfiguration(ObjectMapper mapper, ObjectNode config) throws
       } else {
         config.putPOJO("project", Dtos.asDto(project));
       }
+      config.put("defaultSciName", defaultSciName);
```

Comment:
This can also go in `Constants` if you think it's going to get used anywhere else.

Reply:
I don't think it will.